x-pack/plugins/osquery/cypress/e2e/all/alerts_automated_action_results.cy.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { cleanupRule, loadRule } from '../../tasks/api_fixtures';
+import { checkActionItemsInResults, loadRuleAlerts } from '../../tasks/live_query';
+
+const UUID_REGEX = '[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}';
```

Comment:
I think this can be simplified to 
`/[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}`

and then pass the case insensitive flag to `RegExp()`

`new RegExp(action_id: ${UUID_REGEX},'i')`

Reply:
Ash, I think you can be right, but also when it comes to regex I prefer not to change things that I know that work :P I hope you do not mind :P

Reply:
You can check matching values [here](https://devina.io/regex-data-generator)